components/table/TableModels/RowData.cs

```diff
@@ -112,7 +112,14 @@ public class TableDataItem<TItem> : TableDataItem
         public Table<TItem> Table { get; set; }
 
         public IEnumerable<TItem> Children { get; set; }
-
+        public override bool HasChildren
+        {
+            get
+            {
+                if (Children == null) return false;
+                return Children.Any();
+            }
+        }
```

Comment:
I prefer `public override bool HasChildren => Children?.Any() == true;`

Reply:
> I prefer `public override bool HasChildren => Children?.Any() == true;`

OK，this is better